app/assets/stylesheets/geoblacklight/modules/results.scss

```diff
@@ -26,4 +26,13 @@
 .index_title {
   @extend .text-span;
   @extend .hide-overflow;
+}
+
+.caret-toggle {
+  @extend .fa, .fa-caret-right;
+  width: 10px;
+
```

Comment:
Do we want to add a ``cursor: pointer`` here? I get the select tool when I mouse over the caret which seems a little weird.